### PR TITLE
Add pythonocc

### PR DESCRIPTION
**Pull request by looooo**
A first test. Most likely needs some iterations until it can be merged.

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by looooo** [comment]
@conda-forge/staged-recipes Any ideas how to run tests which are in the $SRC_DIR. I get this error-message:
```
In conda-build 2.1+, the work dir is removed by default before the test scripts run.  You are using the SRC_DIR variable in your test script, but these files have been deleted.  Please see the  documentation regarding the test/source_files meta.yaml section, or pass the --no-remove-work-dir flag.
```

**Comment by chrisburr** [comment]
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section

**Comment by looooo** [comment]
thanks @chrisburr

**Comment by looooo** [comment]
@conda-forge/staged-recipes ready ready for merge.

File: recipes/pythonocc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - cmake
+    - swig >=3.0.11
+  host:
+    - python
+    - occt
+    - tbb-devel
+  run:
+    - python
+    - occt
+    - six
```
Comment (xhochy) [inline recipes/pythonocc/meta.yaml]: Does this package really need six? It is marked as Python 3.7+ only.

**Comment by looooo** [comment]
@xhochy 

the py3.7 build fails with:
`ModuleNotFoundError: No module named 'six'`

In theory older version of python are supported, but I don't want to support py2 and py3.6 can be enabled if anyone needs it.

**Pull request merged**